### PR TITLE
Fix prod aos

### PR DESCRIPTION
**Pull request by Eabramov84**

### CODE REVIEW

File: recipes-domu/domu-image-fusion/files/meta-xt-prod-extra/recipes-core/systemd/systemd_%.bbappend [hunk 1]
```diff
@@ -17,3 +17,20 @@ do_install_append() {
     sed -i -e "s/\/lib\/systemd\/systemd-quotacheck/\/usr\/sbin\/quotacheck -avum/" \
           ${D}${systemd_unitdir}/system/systemd-quotacheck.service
 }
+
+quotacheck_srv = "systemd-quotacheck.service"
+quotaon_srv = "quotaon.service"
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
```
Comment (oleksiimoisieiev) [inline recipes-domu/domu-image-fusion/files/meta-xt-prod-extra/recipes-core/systemd/systemd_%.bbappend]: not needed

File: recipes-domu/domu-image-fusion/files/meta-xt-prod-extra/recipes-core/systemd/systemd_%.bbappend [hunk 2]
```diff
@@ -17,3 +17,20 @@ do_install_append() {
...
+
+do_install_append () {
+    install -d ${D}${sysconfdir}/systemd/system/multi-user.target.wants
+
+    sed -i -e "s/\/lib\/systemd\/systemd-quotacheck/\/usr\/sbin\/quotacheck -avum/" \
```
Comment (oleksiimoisieiev) [inline recipes-domu/domu-image-fusion/files/meta-xt-prod-extra/recipes-core/systemd/systemd_%.bbappend:22]: this line is not needed. Also it should be in one do_install_append line

File: recipes-domu/domu-image-fusion/files/meta-xt-prod-extra/recipes-core/systemd/systemd_%.bbappend [hunk 3]
```diff
@@ -17,3 +17,20 @@ do_install_append() {
...
+         ${D}${systemd_unitdir}/system/${quotacheck_srv}
+
+    sed -i -e "/^.*\/var\/volatile\/log.*$/d" ${D}${sysconfdir}/tmpfiles.d/00-create-volatile.conf
```
Comment (oleksiimoisieiev) [inline recipes-domu/domu-image-fusion/files/meta-xt-prod-extra/recipes-core/systemd/systemd_%.bbappend]: This change is not related to the changes, presented by this commint

File: recipes-domd/domd-image-minimal/files/meta-xt-prod-extra/recipes-connectivity/sja1105-tool/sja1105-tool/sja1105-switch2.service [hunk 4]
```diff
@@ -10,6 +10,8 @@ Type=simple
 # by executing a cold reset via sja1105-tool, ignoring its return value. This
 # way, the SPI interfaces are in a good state again.
 Environment=SJA1105_CONF=/etc/sja1105/sja1105-2.conf
+ExecStartPre=/bin/mkdir -p /var/sja1105
+ExecStartPre=/bin/mkdir -p /var/sja1105
```
Comment (oleksiimoisieiev) [inline recipes-domd/domd-image-minimal/files/meta-xt-prod-extra/recipes-connectivity/sja1105-tool/sja1105-tool/sja1105-switch2.service]: duplicate line

File: recipes-domu/domu-image-fusion/files/meta-xt-prod-extra/recipes-core/systemd/systemd_%.bbappend (hunk 1 above)
Comment (Eabramov84) [reply recipes-domu/domu-image-fusion/files/meta-xt-prod-extra/recipes-core/systemd/systemd_%.bbappend]: accept

File: recipes-domu/domu-image-fusion/files/meta-xt-prod-extra/recipes-core/systemd/systemd_%.bbappend (hunk 2 above)
Comment (Eabramov84) [reply recipes-domu/domu-image-fusion/files/meta-xt-prod-extra/recipes-core/systemd/systemd_%.bbappend:22]: accept

File: recipes-domu/domu-image-fusion/files/meta-xt-prod-extra/recipes-core/systemd/systemd_%.bbappend (hunk 3 above)
Comment (Eabramov84) [reply recipes-domu/domu-image-fusion/files/meta-xt-prod-extra/recipes-core/systemd/systemd_%.bbappend]: accept

**Review by oleksiimoisieiev — APPROVED** [review]
